avocado_vt/plugins/vt_joblock.py

```diff
@@ -68,6 +68,24 @@ def _create_self_lock_file(self, job):
         except Exception as e:
             raise LockCreationError(e)
 
+    def _get_lock_files(self):
+        """
+        Get the list of lock file names under the current lock dir
+
+        :returns: a list with the full path of files that match the
+                  lockfile pattern
+        :rtype: list
+        """
+        try:
+            files = os.listdir(self.lock_dir)
+            pattern = re.compile(r'avocado-vt-joblock-[0-9a-f]{40}-[0-9]+'
+                                 '-[0-9a-z]{8}\.pid')
+            return [os.path.join(self.lock_dir, _) for _ in files
```

Comment:
I'm wondering if `glob` would not suffice. I mean the prefix `avocado-vt-joblock` is IMO sufficient... Anyway we can change this later.